packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -400,14 +390,37 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
     }
 
     return (
-      <div ref={this._root} {...divProps} role={pageElements.length > 0 ? role : undefined} className={css('ms-List', className)}>
-        <div ref={this._surface} className={'ms-List-surface'} role="presentation">
-          {pageElements}
+      <>
+        <div ref={this._root} {...divProps} role={pageElements.length > 0 ? role : undefined} className={css('ms-List', className)}>
+          <div ref={this._surface} className={'ms-List-surface'} role="presentation">
+            {pageElements}
+          </div>
         </div>
-      </div>
+        <EventListener target={window} type="resize" listener={this._onAsyncResize} />
+        <EventListenerWithComputedTargetProps
+          capture
+          resolveTarget={this._getScrollableElement}
+          type="focus"
+          listener={this._onFocus.bind(this)}
+        />
+        <EventListenerWithComputedTargetProps
+          capture
+          resolveTarget={this._getScrollableElement}
+          type="scroll"
+          listener={this._onScroll.bind(this)}
+        />
+        <EventListenerWithComputedTargetProps
+          capture
+          resolveTarget={this._getScrollableElement}
+          type="scroll"
+          listener={this._onAsyncScroll.bind(this)}
```

Comment:
I think we need to update the List tests - they are failing but not for the same reasons as before - those I think were fixed with this new resolveTarget approach but I think now sometimes the component is undefined as a result: https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=54933&view=logs&j=92058b04-f16a-5035-546c-cae3ad5e2f5f&t=17e772b0-f0c3-53ab-7aac-ad6df80e30d7&l=558
![image](https://user-images.githubusercontent.com/4161791/75477985-fa9c6d00-5951-11ea-9d89-c955ca435ca3.png)